core/src/main/resources/hudson/PluginManager/sidepanel_pl.properties

```diff
@@ -0,0 +1,27 @@
+# The MIT License
+#
+# Copyright (c) 2023, Sun Microsystems, Inc., Damian Szczepanik
```

Comment:
This warms my heart (my Sun Microsystems offer letter from 2007 remains a cherished piece of tech memorabilia), but sadly Sun Microsystems ceased to exist as a legal entity about 13 years ago. For new files, the copyright should be assigned solely to you:

```suggestion
# Copyright (c) 2023, Damian Szczepanik
```